optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/AbstractConstraintStreamTest.java

```diff
@@ -74,6 +83,32 @@ public AbstractConstraintStreamTest(boolean constraintMatchEnabled, ConstraintSt
         return scoreDirectorFactory.buildScoreDirector(false, constraintMatchEnabled);
     }
 
+    private static List<Object> removeIndirection(LogicalTuple logicalTuple) {
+        return IntStream.range(0, logicalTuple.getCardinality())
+                .mapToObj(logicalTuple::getItem)
+                .collect(Collectors.toList());
+    }
+
+    private static List<Object> removeIndirection(List<Object> justificationList) {
+        final List<Object> l = justificationList.stream()
```

Comment:
We could as well directly return the `collect()` result without storing it in `l`.